### ISSUE TITLE
Windows reproducible comparison tests failed when rebuild

### ISSUE THREAD

**Issue by sophia-guo** [adoptium/temurin-build#3837]
Windows reproducible comparison tests enabled as dev.system for jdk21+ and failed when rebuilds

https://ci.adoptium.net/job/Test_openjdk21_hs_dev.system_x86-64_windows_testList_0/1/console

```
17:33:18  /usr/bin/rm -f /cygdrive/c/comp-jdk-build/temurin-build/workspace/build/openjdkbuild/make-support/module-deps.gmk
17:33:18  ( printf "DEPS_java.base :=" && /usr/bin/gawk -v MODULE=java.base ' BEGIN      { if (MODULE != "java.base") printf(" java.base"); } /^ *requires/ { sub(/;/, ""); sub(/requires /, " "); sub(/ static /, " "); sub(/ transitive /, " "); sub(/\/\/.*/, ""); sub(/\/\*.*\*\//, ""); gsub(/^ +\*.*/, ""); gsub(/ /, ""); gsub(/\r/, ""); printf(" %s", $0) } END           { printf("\n") }' /cygdrive/c/comp-jdk-build/temurin-build/workspace/build/src/src/java.base/share/classes/module-info.java && printf "TRANSITIVE_MODULES_java.base :=" && /usr/bin/gawk -v MODULE=java.base ' BEGIN      { if (MODULE != "java.base") printf(" java.base"); } /^ *requires  *transitive/ { sub(/;/, ""); sub(/requires/, ""); sub(/transitive/, ""); sub(/\/\/.*/, ""); sub(/\/\*.*\*\//, ""); gsub(/^ +\*.*/, ""); gsub(/ /, ""); gsub(/\r/, ""); printf(" %s", $0) } END           { printf("\n") }' /cygdrive/c/comp-jdk-build/temurin-build/workspace/build/src/src/java.base/share/classes/module-info.java ) >> /cygdrive/c/comp-jdk-build/temurin-build/workspace/build/openjdkbuild/make-support/module-deps.gmk 
17:33:18  ( printf "DEPS_java.compiler :=" && /usr/bin/gawk -v MODULE=java.compiler ' BEGIN      { if (MODULE != "java.base") printf(" java.base"); } /^ *requires/ { sub(/;/, ""); sub(/requires /, " "); sub(/ static /, " "); sub(/ transitive /, " "); sub(/\/\/.*/, ""); sub(/\/\*.*\*\//, ""); gsub(/^ +\*.*/, ""); gsub(/ /, ""); gsub(/\r/, ""); printf(" %s", $0) } END 
...
build/workspace/build/src/test/jaxp_JTREG_TEST_GROUPS := " ; /usr/bin/sed -n -e 's/^\#.*//g' -e 's/\([^ ]*\)\w*=.*/\1/gp' /cygdrive/c/comp-jdk-build/temurin-build/workspace/build/src/test/jaxp/TEST.groups | /usr/bin/sort -u | /usr/bin/tr '\n' ' ' ;   printf "\n/cygdrive/c/comp-jdk-build/temurin-build/workspace/build/src/test/lib-test_JTREG_TEST_GROUPS := " ; /usr/bin/sed -n -e 's/^\#.*//g' -e 's/\([^ ]*\)\w*=.*/\1/gp' /cygdrive/c/comp-jdk-build/temurin-build/workspace/build/src/test/lib-test/TEST.groups | /usr/bin/sort -u | /usr/bin/tr '\n' ' ' ;  \
17:33:26  ) > /cygdrive/c/comp-jdk-build/temurin-build/workspace/build/openjdkbuild/make-support/find-tests.gmk
17:33:26  printf "\nMAKE_TEST_TARGETS := " >> /cygdrive/c/comp-jdk-build/temurin-build/workspace/build/openjdkbuild/make-support/find-tests.gmk
17:33:26  /usr/bin/make -s --no-print-directory  -r -R -I /cygdrive/c/comp-jdk-build/temurin-build/workspace/build/src/make/common SPEC=/cygdrive/c/comp-jdk-build/temurin-build/workspace/build/openjdkbuild/spec.gmk MAKE_LOG_FLAGS=""  \
17:33:26      SPEC=/cygdrive/c/comp-jdk-build/temurin-build/workspace/build/openjdkbuild/spec.gmk -f /cygdrive/c/comp-jdk-build/temurin-build/workspace/build/src/test/make/TestMake.gmk print-targets \
17:33:26      TARGETS_FILE=/cygdrive/c/comp-jdk-build/temurin-build/workspace/build/openjdkbuild/make-support/find-tests.gmk
17:33:27  make[7]: Leaving directory '/cygdrive/c/comp-jdk-build/temurin-build/workspace/build/src'
17:33:27  make[6]: Nothing to be done for 'product-images'.
17:33:27  make[6]: Nothing to be done for 'legacy-jre-image'.
17:33:27  make[6]: Nothing to be done for 'test-image'.
17:33:27  make[6]: Nothing to be done for 'static-libs-image'.
17:33:27  make[6]: Leaving directory '/cygdrive/c/comp-jdk-build/temurin-build/workspace/build/openjdkbuild'
17:33:27  build.sh : 21:33:26 : Build complete ...
17

```

**Comment by sophia-guo** [comment]
SBOM format updated https://github.com/adoptium/temurin-build/pull/3835, which may also affect windows reproducible parsing functions. Rebuild windows jdk with  https://github.com/adoptium/temurin-build/pull/3835 in  https://ci.adoptium.net/job/build-scripts/job/jobs/job/jdk21u/job/jdk21u-windows-x64-temurin/116/.  When the new jdk sbom available, will manually trigger a window reproducible tests to see the result.

**Comment by sophia-guo** [comment]
```
11:17:11  Files /cygdrive/c/comp-jdk-build/compare/src_jdk/bin/server/jvm.dll and /cygdrive/c/comp-jdk-build/compare/tar_jdk/bin/server/jvm.dll differ
11:17:11  Files /cygdrive/c/comp-jdk-build/compare/src_jdk/jmods/expanded_java.base.jmod/lib/server/jvm.dll and /cygdrive/c/comp-jdk-build/compare/tar_jdk/jmods/expanded_java.base.jmod/lib/server/jvm.dll differ
11:17:11  Number of differences: 2
```

No issue with rebuild. Two differences. Close this one, the original error may be fixed by #3835 .
